Dockerfile

```diff
@@ -1,35 +1,27 @@
-FROM ubuntu-debootstrap:14.04
+FROM php:5.6-alpine
 MAINTAINER Christian Lück <christian@lueck.tv>
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
-  nginx supervisor php5-fpm php5-cli \
-  php5-pgsql php5-mysql php5-sqlite php5-mssql \
-  wget
+ENV ADMINER_VERSION 4.2.4
```

Comment:
dunno. changed in 0fb8d51a41d56b5aaa84862054b7c22095a050bc anyway.

in fact `ADD` directive will *always* download fresh version on build command (doesn't use cached one), so maybe indeed it would be good idea to get latest...